atlasdb-tests-shared/src/main/java/com/palantir/atlasdb/timestamp/TimestampServiceTests.java

```diff
@@ -0,0 +1,141 @@
+/**
+ * Copyright 2017 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.timestamp;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.greaterThan;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.lessThan;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.concurrent.ConcurrentSkipListSet;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+import org.assertj.core.api.Assertions;
+import org.hamcrest.MatcherAssert;
+
+import com.palantir.timestamp.TimestampMigrationService;
+import com.palantir.timestamp.TimestampRange;
+import com.palantir.timestamp.TimestampService;
+
+public class TimestampServiceTests {
+    private static final long ONE_MILLION = 1000 * 1000;
+    private static final long TWO_MILLION = 2 * ONE_MILLION;
+
+    private static ExecutorService executor = Executors.newFixedThreadPool(16);
+
+    private TimestampServiceTests() {
+
+    }
+
+    public static void timestampsAreReturnedInOrder(TimestampService timestampService) {
+        List<Long> timestamps = new ArrayList<>();
+
+        timestamps.add(timestampService.getFreshTimestamp());
+        timestamps.add(timestampService.getFreshTimestamp());
+
+        MatcherAssert.assertThat(timestamps.get(0), lessThan(timestamps.get(1)));
```

Comment:
We normally static import the Hamcrest/AssertJ `assertThat` methods as well. It seems you've copied this over, so possibly not the scope of this PR, but we should fix it soon.